ras_party/controllers/account_controller.py

```diff
@@ -135,7 +130,7 @@ def post_respondent(party, tran, session):
         raise RasError("Error during enrolment process", status=500)
 
     register_user(party, tran)
-
+    disable_iac(enrolment_code=party['enrolmentCode'])
```

Comment:
- Now that the session is committed above, we should stop passing `tran` to `register_user` and remove the `dummy_compensating_action` code from that method.
- What should the behaviour be if we have successfully enrolled and registered a user, but disabling the IAC fails? I think currently this will create a 500 error for the user.

Reply:
@ajmaddaford I spoke to Dave and he basically just said log that the iac has failed to be deactivated using the case id and a splunk alert should probably be set up to catch this. Do we want the process to carry on after as the respondent will have been added to django and party?

Reply:
👍 that sounds reasonable